pkgs/by-name/xc/xcbeautify/package.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, unzip, lib }:
+
+let
+  inherit (lib) getAttr;
+in
+stdenv.mkDerivation rec {
+  pname = "xcbeautify";
+  version = "1.4.0";
+  arch = if stdenv.hostPlatform.isAarch64 then "arm64" else "x86_64";
+
+  src = fetchurl {
+    url = "https://github.com/cpisciotta/xcbeautify/releases/download/${version}/xcbeautify-${version}-${arch}-apple-macosx.zip";
+    sha256 = getAttr arch {
```

```suggestion
  src = fetchurl {
    url = "https://github.com/cpisciotta/xcbeautify/releases/download/${version}/xcbeautify-${version}-${stdenv.hostPlatform.darwinArch}-apple-macosx.zip";
    sha256 = lib.getAttr stdenv.hostPlatform.darwinArch {
```

`stdenv.hostPlatform.darwinArch` matches the format you need.